manual/jsdoc/AGGrid.html

```diff
@@ -3474,7 +3474,7 @@ <h4 class="name" id="getSelectedRowIndex"><span class="type-signature"></span>ge
 
 <div class="description usertext">
     Return the index of the selected row.  <code>null</code> is returned if no row is selected.
-If multiple rows are selected, the index of the forst selected row is returned.
+If multiple rows are selected, the index of the first selected row is returned.
```

Comment:
this typo is present in a number of locations. I didn't see any particular indication that it was intentional as opposed to copy+paste.